test/extended/templates/helpers.go

```diff
@@ -58,3 +78,95 @@ func setUser(cli *exutil.CLI, user *userapi.User) {
 		cli.ChangeUser(user.Name)
 	}
 }
+
+func EnsureTSB(tsbOC *exutil.CLI) (osbclient.Client, *exec.Cmd) {
+	exists := true
+	if _, err := tsbOC.AdminKubeClient().Extensions().DaemonSets(tsbNS).Get("apiserver", metav1.GetOptions{}); err != nil {
+		if !kerrors.IsNotFound(err) {
+			o.Expect(err).NotTo(o.HaveOccurred())
+		}
+		exists = false
+	}
+
+	if !exists {
+		e2e.Logf("Installing TSB onto the cluster for testing")
+		_, _, err := tsbOC.AsAdmin().WithoutNamespace().Run("create", "namespace").Args(tsbNS).Outputs()
```

Comment:
do we run into trouble here if multiple tests run in parallel and run EnsureTSB at the same time?